ledger/participant-integration-api/src/main/scala/platform/store/FlywayMigrations.scala

```diff
@@ -70,5 +70,5 @@ class FlywayMigrations(jdbcUrl: String)(implicit logCtx: LoggingContext) {
 
 object FlywayMigrations {
   def configurationBase(dbType: DbType): FluentConfiguration =
-    Flyway.configure().locations("classpath:db/migration/" + dbType.name)
+    Flyway.configure().locations("classpath:com/daml/platform/db/migration/" + dbType.name)
```

Comment:
I guess this classpath location is for both the scala and the sql migrations?
If we want to keep omitting the `com/daml` prefix we could move them to `platform/db/migration` (or keep them at `db/migration`) and add a second location element here.

Reply:
Can we move migrations? I thought they had to stay put.

@rautenrieth-da, did you test that you can re-run the migrations on an already-existing database created with `master`?

Reply:
I have checked manually that I can run daml-on-sql from master, add some data to it, then run daml-on-sql from this branch against the same postgres index DB.

It's a very good point though, so I looked further into this:
Here's an excerpt from the `flyway_schema_history` table. 
```
version | type |                         script                         |  checksum   
---------+------+--------------------------------------------------------+-------------
 1       | SQL  | V1__Init.sql                                           |   -72965523
 2.0     | SQL  | V2_0__Contract_divulgence.sql                          |  1461208192
 2.1     | JDBC | db.migration.postgres.V2_1__Rebuild_Acs                |            
 3       | JDBC | db.migration.postgres.V3__Recompute_Key_Hash           |            
 4.0     | SQL  | V4_0__Add_parties.sql                                  |  2091746815
 4.1     | JDBC | db.migration.postgres.V4_1__Collect_Parties            |            
 5       | SQL  | V5__Add_packages.sql                                   |  1089030748
 6       | SQL  | V6__External_Ledger_Offset.sql                         |  1642074148
 7       | SQL  | V7__Command_deduplication.sql                          |   478192496
 8       | SQL  | V8__Contract_Divulgence.sql                            | -1984400579
 9       | SQL  | V9__Contract_Divulgence.sql                            |   -39708021
 10.0    | SQL  | V10_0__Extract_Event_Data.sql                          |  1741793687
 10.1    | JDBC | db.migration.postgres.V10_1__Populate_Event_Data       |            
 10.2    | SQL  | V10_2__Extract_Event_Data.sql                          | -1727099174
```

Manually changing the "script" column doesn't affect anything. Manually changing the checksum or the version however breaks the database.